src/expr/skolem_manager.h

```diff
@@ -231,7 +233,8 @@ class SkolemManager
    */
   Node mkSkolemFunction(SkolemFunId id,
                         TypeNode tn,
-                        Node cacheVal = Node::null());
+                        Node cacheVal = Node::null(),
+                        int flags = NodeManager::SKOLEM_DEFAULT);
```

Comment:
use unsigned?

Reply:
This is to match `NodeManager::` flags.  We shouldn't mix types here until those flags are fully refactored at once. (Note that `int flags` appears as arguments to other methods in this class for the same reason.

Reply:
I see, sounds good